engine/installation/linux/docker-ce/centos.md

```diff
@@ -77,31 +77,38 @@ from the repository.
     ```
 
 2.  Use the following command to set up the **stable** repository. You always
-    need the **stable** repository, even if you want to install **edge** builds
-    as well.
+    need the **stable** repository, even if you want to install builds from the
+    **edge** or **testing** repositories as well.
```

Comment:
Do we really want people to only have Edge and not Stable at all? I think maybe we should think it through and  change it after release.